app/services/idv/steps/in_person/address_step.rb

```diff
@@ -18,8 +18,11 @@ def analytics_submitted_event
 
         def call
           Idv::InPerson::AddressForm::ATTRIBUTES.each do |attr|
-            next if attr == :same_address_as_id && capture_secondary_id_enabled?
-            flow_session[:pii_from_user][attr] = flow_params[attr]
+            if capture_secondary_id_enabled? && updating_address? && attr == :same_address_as_id
+              flow_session[:pii_from_user][:same_address_as_id ] = 'false'
```

Comment:
spacing
```suggestion
              flow_session[:pii_from_user][:same_address_as_id] = 'false'
```

Reply:
fixed in 5dc54989353985012ffee7aa7ebf4daa81bdf326